Description: Fix FTBFS on bionic. Compiler errors:
 In file included from Unified_cpp_widget_gtk1.cpp:2:
 /<<BUILDDIR>>/firefox-92.0~b2+build1/widget/gtk/WaylandBuffer.cpp:261:39: error: unknown type name 'GLContext'; did you mean 'EGLContext'?
    const LayoutDeviceIntSize& aSize, GLContext* aGL) {
                                      ^~~~~~~~~

Author: Rico Tzschichholz <ricotz@ubuntu.com>

diff --git a/widget/gtk/WaylandBuffer.cpp b/widget/gtk/WaylandBuffer.cpp
index f3fc4093623f..78d213d5d302 100644
--- a/widget/gtk/WaylandBuffer.cpp
+++ b/widget/gtk/WaylandBuffer.cpp
@@ -197,7 +197,7 @@ void WaylandBufferSHM::DumpToFile(const char* aHint) {
 
 /* static */
 RefPtr<WaylandBufferDMABUF> WaylandBufferDMABUF::Create(
-    const LayoutDeviceIntSize& aSize, GLContext* aGL) {
+    const LayoutDeviceIntSize& aSize, gl::GLContext* aGL) {
   RefPtr<WaylandBufferDMABUF> buffer = new WaylandBufferDMABUF(aSize);
 
   const auto flags =
